src/CentreonRemote/Application/Clapi/CentreonRemoteServer.php

```diff
@@ -67,6 +71,10 @@ public function disableRemote()
         $result = $this->getDi()['centreon.db-manager']->getRepository(InformationsRepository::class)->toggleRemote('no');
         echo 'Done'. "\n";
 
+        echo "\n Restore 'central' instance type...";
+        system("sed -i -r 's/remote/central/g' " . _CENTREON_ETC_ . "/conf.pm");
```

Comment:
Sed should be more restrictive as the word "remote" can be used for password or database name or anything else.